tests/driver.sh

```diff
@@ -21,5 +21,9 @@ fi
 $TESTSRV -q -a 127.0.0.1 &
 echo $! >> $TESTPID
 sleep 1
-python3 -m unittest $1.py
+if [[ $# -eq 1 ]] ; then
```

Comment:
I did this change to give the chance to call single methods of the test suite, since I have all utf8 tests (which are a lot) in a single py file